cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/NativeUtils.java

```diff
@@ -149,6 +209,37 @@ public static void createReflectConfig(Path nativeConfigPath, Package currentPac
         }
     }
 
+    //Create $ORIG_function methods class mapping
+    private static void extractMockFunctionClassMapping(Map<String, TestSuite> testSuiteMap,
+                                                        Map<String, List<String>> mockFunctionClassMapping) {
+        for (Map.Entry<String, TestSuite> testSuiteEntry : testSuiteMap.entrySet()) {
+            TestSuite suite = testSuiteEntry.getValue();
+            for (Map.Entry<String, String> mockFunctionEntry : suite.getMockFunctionNamesMap().entrySet()) {
+                String key = mockFunctionEntry.getKey();
+                String functionToMockClassName;
+                String functionToMock;
+                if (key.indexOf(MOCK_LEGACY_DELIMITER) == -1) {
+                    functionToMockClassName = key.substring(0, key.indexOf(MOCK_FN_DELIMITER));
+                    functionToMock = key.substring(key.indexOf(MOCK_FN_DELIMITER)+1);
```

Comment:
```suggestion
                    functionToMock = key.substring(key.indexOf(MOCK_FN_DELIMITER) + 1);
```
can be applied to lines below as well